frontend/src/individual_sellingform.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+
+import { makeStyles, TextField } from "@material-ui/core";
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from "@material-ui/pickers";
+import { Autocomplete } from "@material-ui/lab";
+import DateFnsUtils from "@date-io/date-fns";
+
+import {
+  Genebank,
+  herdLabel,
+  Individual,
+  LimitedHerd,
+} from "./data_context_global";
+
+const useStyles = makeStyles({
+  inputField: {
+    margin: "1em 0",
+  },
+});
+
+export function IndividualSellingform({
```

Comment:
```suggestion
export function IndividualSellingForm({
```

Reply:
This change makes sense, of course. Renamed it.